src/DataPolicyTranslationHandler.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Drupal\gdpr_consent;
+
+use Drupal\content_translation\ContentTranslationHandler;
+
+/**
+ * Defines the translation handler for data_policy.
+ */
+class DataPolicyTranslationHandler extends ContentTranslationHandler {
```

Comment:
@OH17L this class does nothing. Shouldn't we just point to the normal class? E.g. `ContentTranslationHandler`?

Reply:
@bramtenhove, I will check it.